docs/source/installation.rst

```diff
@@ -100,98 +100,270 @@ Step 1: Setup dedicated python environment
             If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_.
 
 
-Step 2: Install ``ivadomed``
-++++++++++++++++++++++++++++
-
+Step 2: Install ``ivadomed`` with GPU or CPU Support
+++++++++++++++++++++++++++++++++++++++++++++++++++++
 
     .. tabs::
+        
+        .. tab:: NVIDIA GPU Support
+
+            ``ivadomed`` requires a minimum PyTorch version of either
+            1.8.1 or 1.8.2 which supports CUDA 10.2 and CUDA 11.1 builds
+            by default. 
+            
+            Ampere-based GPUs (with a Compute Capability of 8.x) natively
+            support CUDA>=11.1. Although CUDA 11.1 is backward compatible
+            but CUDA 10.2 is preferred for older GPUs.
```

Comment:
Interesting updates #1141 & the upstream torch thread, at least for torch ~=1.8.1